applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -440,21 +496,21 @@ static void scan_init(void)
 		return;
 	}
 
-	static const struct bt_le_scan_param sp = {
+	const struct bt_le_scan_param sp = {
```

Comment:
why to remove static?

Reply:
The nrf BLE scanning module makes copy of the parameters that are passed here.
I wanted to indicate that these structures are no longer in use after the scanning module is initialized.

Reply:
do you mean NCS library?

using static const here makes object placed in flash; if you think about what `const static` indicates, it often marks a constant data used for initialization; it is not really important if it is used once or more in runtime.
When you remove const I automatically looked where this object is modified.

What scanning library is doing depends on its API; I don't see how our way of storing initialization data helps understand what library is doing; 

Sometimes libraries cast away const for some reason, if this is not the case please keep constant initialization data `static const`.

Reply:
Ok, I will switch back to `static const` here.